Use dump-as-markup.js in fast/attachment/attachment-select-on-click.html

rniwa opened the pull request:
#### e8b876bcb7bbea4a163f855acb78f8f3bad87460
<pre>
Use dump-as-markup.js in fast/attachment/attachment-select-on-click.html
<a href="https://bugs.webkit.org/show_bug.cgi?id=249121">https://bugs.webkit.org/show_bug.cgi?id=249121</a>

Reviewed by Darin Adler.

This test is testing selection across attachment element. Use dump-as-markup.js for clarity.

* LayoutTests/fast/attachment/attachment-select-on-click-expected.txt: Added.
* LayoutTests/fast/attachment/attachment-select-on-click.html:
* LayoutTests/platform/gtk/fast/attachment/attachment-select-on-click-expected.txt: Removed.
* LayoutTests/platform/ios/fast/attachment/attachment-select-on-click-expected.txt: Removed.
* LayoutTests/platform/mac/fast/attachment/attachment-select-on-click-expected.png: Removed.
* LayoutTests/platform/mac/fast/attachment/attachment-select-on-click-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/attachment/attachment-select-on-click-expected.txt: Removed.
* LayoutTests/platform/wpe/fast/attachment/attachment-select-on-click-expected.txt: Removed.

Canonical link: <a href="https://commits.webkit.org/257768@main">https://commits.webkit.org/257768@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/2c0b106bf3088a15e0fc558a1c730032cc305d37

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/99925 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/9094 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/33004 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/109275 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/169512 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/9975 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/86388 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/92373 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/107178 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/105692 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/7531 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/3/builds/90812 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/34252 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/89437 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/22203 "Passed tests") | [⏳ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/API-Tests-GTK-EWS "Waiting to run tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/2895 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/23716 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/2852 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/46088 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/5333 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/8984 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/43190 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/4703 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

darinadler commented on LayoutTests/fast/attachment/attachment-select-on-click.html:
```diff
@@ -17,8 +18,9 @@
-    document.getElementById("result").innerText = 'Start offset ' + range.startOffset + ' End offset ' + range.endOffset;
+    Markup.dump(container);
+//    var range = window.getSelection().getRangeAt(0);
+//    document.getElementById("result").innerText = 'Start offset ' + range.startOffset + ' End offset ' + range.endOffset;
```
Why keep this commented out code?

rniwa replied:
Oh weird. I must have forgotten to commit --amend.

darinadler commented on LayoutTests/fast/attachment/attachment-select-on-click.html:
```diff
@@ -17,8 +18,9 @@
     eventSender.mouseUp(0);
-    var range = window.getSelection().getRangeAt(0);
-    document.getElementById("result").innerText = 'Start offset ' + range.startOffset + ' End offset ' + range.endOffset;
+    Markup.dump(container);
```
Where is the new expected output file?

rniwa committed:
Use dump-as-markup.js in fast/attachment/attachment-select-on-click.html
https://bugs.webkit.org/show_bug.cgi?id=249121

Reviewed by Darin Adler.

This test is testing selection across attachment element. Use dump-as-markup.js for clarity.

* LayoutTests/fast/attachment/attachment-select-on-click-expected.txt: Added.
* LayoutTests/fast/attachment/attachment-select-on-click.html:
* LayoutTests/platform/gtk/fast/attachment/attachment-select-on-click-expected.txt: Removed.
* LayoutTests/platform/ios/fast/attachment/attachment-select-on-click-expected.txt: Removed.
* LayoutTests/platform/mac/fast/attachment/attachment-select-on-click-expected.png: Removed.
* LayoutTests/platform/mac/fast/attachment/attachment-select-on-click-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/attachment/attachment-select-on-click-expected.txt: Removed.
* LayoutTests/platform/wpe/fast/attachment/attachment-select-on-click-expected.txt: Removed.

Canonical link: https://commits.webkit.org/257768@main
